[DOC] Update homepage > "about tedana" to redirect readers to relevant page

monicayao opened the pull request:
<!---
This is a suggested pull request template for tedana.
It's designed to capture information we've found to be useful in reviewing pull requests.

If there is other information that would be helpful to include, please don't hesitate to add it!

Please also label your pull request with the relevant tags.
See here for more information and a list of available options:
https://github.com/ME-ICA/tedana/blob/master/CONTRIBUTING.md#pull-requests
-->

<!-- Please indicate after the # which issue you're closing with this PR.
This is helpful for the maintainers AND will magically close the issue when this
pull request is merged!
https://help.github.com/articles/closing-issues-using-keywords -->
Addresses issue #290 

<!-- Please give a brief overview of what has changed in the PR.
If you're not sure what to write, consider it a note to the maintainers to indicate
what they should be looking for when they review the pull request. -->
Changes proposed in this pull request:

- This is just a brief update to the homepage file
- In the about section, it redirects the users to other pages depending on what information they are looking for

tsalo commented on docs/index.rst:
```diff
@@ -51,8 +51,15 @@ The ME-ICA pipeline originally performed both pre-processing and TE-dependent
+visit `Multi-echo fMRI`_. 
+
+For a detailed procedure of how ``tedana`` analyzes the data from multi-echo fMRI, 
+visit `Processing pipline details`_.
```
pipline --> pipeline

tsalo commented on docs/index.rst:
```diff
@@ -51,8 +51,15 @@ The ME-ICA pipeline originally performed both pre-processing and TE-dependent
 
+.. _ME-ICA: https://github.com/me-ica/me-ica
+.. _Multi-echo fMRI: https://tedana.readthedocs.io/en/latest/multi-echo.html
+.. _Processing pipline details: https://tedana.readthedocs.io/en/latest/multi-echo.html
```
This points to the multi-echo page, not the approach page.

monicayao replied:
Apologies!

jbteves commented on docs/index.rst:
```diff
@@ -51,8 +51,15 @@ The ME-ICA pipeline originally performed both pre-processing and TE-dependent
+visit `Multi-echo fMRI`_. 
+
+For a detailed procedure of how ``tedana`` analyzes the data from multi-echo fMRI, 
+visit `Processing pipline details`_.
```
```suggestion
visit `Processing pipeline details`_.
```

jbteves replied:
@monicayao you can just commit the above suggestion instead of manually editing it.

jbteves commented on docs/index.rst:
```diff
@@ -51,8 +51,15 @@ The ME-ICA pipeline originally performed both pre-processing and TE-dependent
 working with data which has been previously preprocessed.
 
-.. _ME-ICA: https://github.com/me-ica/me-ica
+For a summary of multi-echo fMRI, which is the technology ``tedana`` builds on, 
```
I feel like multi-echo is more of a technique than a technology, but I can be convinced otherwise.

monicayao replied:
I wrote this while looking at this [page](https://www.sciencedirect.com/science/article/pii/S1053811917302410), which says 

> An emerging fMRI technology, multi-echo (ME)-fMRI, makes fMRI more interpretable by enabling the detection of BOLD or artifact origins of acquired signals.

So I was thinking that I could call ME-fMRI "technology." Otherwise, I do agree that multi-echo is a technique used in fMRI. What are your thoughts?

jbteves replied:
Hm, @emdupre @tsalo @dowdlelt is "technology" the preferred etymology? If so then disregard my comment.

tsalo replied:
I vote for "imaging technique".

monicayao commented:
> This looks good once those typos are corrected. Additionally, could you edit your PR title to state what the small changes are? Try to keep it to a very small number of characters (<80) including the [DOC] prefix. It is often surprisingly difficult to articulate these things compactly, so if you would like my input feel free to ping me.

@jbteves I've changed the title of the PR, but it still seems quite long. How can I compress it without making it lose its meaning?

jbteves commented:
@monicayao you still have another 35 characters to add words, I don't think there's a need to compress; on the contrary, I want you to add more. I just didn't want it to be more than 80 when you were done. Sorry, that was probably not clear.

monicayao commented:
@jbteves sorry for being offline for these two days, I had to do more house work but I am ready to make a lot of changes tonight! I have changed the title of the PR again and changed technology to imaging technique as per Taylor's suggestion 😄

monicayao commented:
@jbteves Thank you! But I can't merge because I don't have write access 😓

jbteves commented:
... ah. That would do it. I’ll go ahead and merge it.

jbteves commented:
@allcontributors add @monicayao for documentation
